src/routes/form/+page.svelte

```diff
@@ -23,4 +25,61 @@
 		<option value="9">Sem IX</option>
 		<option value="10">Sem X</option>
 	</select>
+	<label for="role">Roll No</label>
+	<input type = "text" id = "role" name = "role">
+	<label for="category">Category</label>
+	<select id="category">
+		<option value="1">General</option>
+		<option value="2">EWS</option>
+		<option value="3">OBC</option>
+		<option value="4">WDP</option>
+		<option value="5">SC</option>
+		<option value="6">ST</option>
+		<option value="7">VHC</option>
+		<option value="8">PHC</option>
+	</select>
+	<label for="phone">Phone No</label>
+	<input type = "tel" id = "phone" name = "phone">
+
+	
+	<strong>Details of the fees payment</strong>
+	<br>
+	<label for="acc-holder">Name of the account holder</label>
+	<input type="text" id="acc-holder" name="acc-holder">
+	<label for="acc-no">Account No</label>
+	<input type="number" id="acc-no" name="acc-no">
+	<label for="bank">Name of the bank</label>
+	<input type="text" id="bank" name="bank">
+	<label for="branch">Branch</label>
+	<input type="text" id="branch" name="branch">
+
+	<label for="ifsc">IFSC code</label>
+	<input type="number" id="ifsc" name="ifsc">
```

Comment:
ifsc can't be number :joy:  first four characters are bank code and they are letters

Reply:
Did not know that. Will change